TACC/Core-Portal
# Task/fp 273 app height too short in safari browser

## Overview: ##

## PR Status: ##

* Work in Progress.
* Hold.

## Related Jira tickets: ##

* [FP-273](https://jira.tacc.utexas.edu/browse/FP-273)

## Summary of Changes: ##

1. Remove redundant style (duplicate of style removed in change 2).
1. Remove style that is ineffectual in Safari.
1. Extend use flex solution, which works in Safari too.

## Testing Steps: ##
1. As logged in user, load portal.
2. Check that, on each browser, the boxes of content, within each section, stretch to fill available space:
    - Browsers:
    - Sections:
        - [x] Data Files
        - [x] Applications
        - [x] Allocations

## Notes: ##
I should have continued this “flexible” solution down another HTML layer when I first introduced it; but, I was trying to introduce minimal code, and I didn’t check Safari.
